geomstats/geometry/lie_algebra.py

```diff
@@ -10,7 +10,7 @@
 
 BCH_INFO = gs.asarray([
     [int(x) for x in i.strip().split()]
-    for i in open("geomstats/geometry/bch_coefficients.dat").readlines()
+    for i in open("/geomstats/geometry/bch_coefficients.dat").readlines()
```

Comment:
Remove the `/`, we need a relative path here, otherwise it is dependent on the user's file organization.